### PR TITLE
Add documentation for the package_version gatherer

### PR DESCRIPTION
**Pull request by rtorrero**
This PR simply adds a bit of documentation to the newly merged package_version gatherer.
- I've followed @arbulu89 suggestions on remove some unnecessary stuff (see pr #89)
- I've tweaked a bit the other corosync table to improve readability when not rendered somewhere without markdown support

### CODE REVIEW

**Review by nelsonkopliku — APPROVED** [review]

File: guides/gatherers.md [hunk 1]
```diff
@@ -66,3 +67,30 @@ facts:
...
+Sample arguments
+| Name                   | Return value  
+| ---------------------- | -------------------------------------------------------------------------------------
+| `package_name`         | a string containing the installed version for the package `package_name`, e.g "2.4.5"
+
+Specification examples:
+
+```yaml
+facts:
+  - name: package_version-corosync
```
Comment (arbulu89) [inline guides/gatherers.md:85]: Come on man! Let's avoid having this `_-` thins things in the examples! XD
Put underscore only
They are ugly as hell and people take reference on them

File: guides/gatherers.md [hunk 2]
```diff
@@ -66,3 +67,30 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+
+### package_version
+
+This gatherer allows checking the installation of a certain package in the system
```
Comment (arbulu89) [inline guides/gatherers.md:74]: I would rephrase to a more specific description.
`This gatherer returns the version as a string of the specified package` or similar.

File: guides/gatherers.md [hunk 3]
```diff
@@ -66,3 +67,30 @@ facts:
...
+
+Sample arguments
```
Comment (arbulu89) [inline guides/gatherers.md:76]: I would sincerely remove this `sample arguments` entry. It is quite obvious how to use it in the further examples.

**Review by arbulu89 — CHANGES_REQUESTED** [review]

File: guides/gatherers.md (hunk 1 above)
Comment (rtorrero) [reply guides/gatherers.md:85]: It's not random! probably has a bit of OCD though :smile: .I want to make a clear difference between the name of the gatherer vs the argument we are giving

File: guides/gatherers.md (hunk 3 above)
Comment (rtorrero) [reply guides/gatherers.md:76]: I agree that for this gatherer, it's quite obvious, but having consistency between all gatherers sections is important IMO. It barely makes a difference in space, and consistent sections generally allow for faster reading and finding what you want.

File: guides/gatherers.md (hunk 1 above)
Comment (arbulu89) [reply guides/gatherers.md:85]: I would still change it. It is obvous that we are talking about this gatherer as the `gatherer` field already says so.
I would be fine if you change to `corosync_version`.

File: guides/gatherers.md (hunk 3 above)
Comment (arbulu89) [reply guides/gatherers.md:76]: So put some real package name, if its an example.
I don't know, I find it so redundant...

File: guides/gatherers.md (hunk 1 above)
Comment (fabriziosestito) [reply guides/gatherers.md:85]: +1 to remove the dash and just use the lowdash for consistency. `corosync_version` or `corosync_package_version` will do

Comment (fabriziosestito) [reply guides/gatherers.md:85]: @rtorrero I just realized it breaks the evaluation language as `-`  cannot appear in the variables: https://rhai.rs/book/ref/variables.html#valid-names
